reco_utils/dataset/movielens.py

```diff
@@ -129,6 +129,47 @@ def item_has_header(self):
 ERROR_LOCAL_CACHE_PATH = """Local cache path only accepts a zip file path:
     use/something/like_this.zip"""
 
+def load_pandas_df_from_ds(
```

Comment:
I don't understand very well the need of this function, if you just want to load a dataframe from a file, you can directly use pd.read_csv in the code. Is that what you want?

Reply:
Yes it's basically just pd.read_csv(). It's in movielens.py because of contance references. If I try to call pd.read_csv() in the notebook, it will be quite lengthy.